File: code-coaches/easy/candles/Candles.java
import java.util.Scanner;

public class Candles {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int numFriends = input.nextInt();
		int candlesPerPerson = 9;
		// + candlesPerPerson because we order 9 candles, independently of the number of friends
		int candles = numFriends * candlesPerPerson + candlesPerPerson;

		System.out.print(candles);

		input.close();
	}

}

File: code-coaches/easy/candles/candles.py
def main():
    num_friends = int(input())
    candles_per_person = 9
    # + candlesPerPerson because we order 9 candles, independently of the number of friends
    candles = num_friends * candles_per_person + candles_per_person
    print(candles)


if __name__ == "__main__":
    main()
